app/Models/TagDAO.php

```diff
@@ -328,11 +328,14 @@ public function tagEntry($id_tag, $id_entry, $checked = true) {
 		}
 	}
 
-	public function getTagsForEntry($id_entry) {
+	public function getTagsForEntry($id_entry, $onlyCheckedTags = false) {
 		$sql = 'SELECT t.id, t.name, et.id_entry IS NOT NULL as checked '
 			 . 'FROM `_tag` t '
-			 . 'LEFT OUTER JOIN `_entrytag` et ON et.id_tag = t.id AND et.id_entry=? '
-			 . 'ORDER BY t.name';
+			 . 'LEFT OUTER JOIN `_entrytag` et ON et.id_tag = t.id AND et.id_entry=? ';
+		if ($onlyCheckedTags) {
+			$sql .= 'WHERE checked = true ';
```

Comment:
That query is a bit strange. Only tested in PosgreSQL, where it does not work:

```
SQL error getTagsForEntry: ERROR: column "checked" does not exist
```

I have not investigated, but I believe this condition needs to be placed before the join.